docs/BR.md

```diff
@@ -994,25 +1026,88 @@ Databases maintained by the CA, its owner, or its affiliated companies do not qu
+
+#### 3.2.2.9 Multi-Perspective Issuance Corroboration
+
+Multi-Perspective Issuance Corroboration attempts to corroborate the determinations (i.e., domain validation pass/fail, CAA permission/prohibition) made by the Primary Network Perspective from multiple remote Network Perspectives before Certificate issuance. This process can improve protection against equally-specific prefix Border Gateway Protocol (BGP) attacks or hijacks.
+
+Each corroborating Network Perspective's response MUST provide the CA with the necessary information to allow it to affirmatively assess:
+- a. the presence of the expected 1) Random Value, 2) Request Token, 3) IP Address, or 4) Contact Address, as required by the relied upon validation method specified in Sections 3.2.2.4 and 3.2.2.5; and
+- b. the CA's authority to issue to the requested domain(s), as specified in Section 3.2.2.8.
+
+[Section 3.2.2.4](#3224-validation-of-domain-authorization-or-control) and [Section 3.2.2.5](#3225-authentication-for-an-ip-address) describe the validation methods that require the use of Multi-Perspective Issuance Corroboration and how a Network Perspective can corroborate the outcomes determined by the Primary Network Perspective. 
+
+Results or information obtained from one Network Perspective MUST NOT be reused or cached when performing validation through subsequent Network Perspectives (e.g., different Network Perspectives cannot rely on a shared DNS cache to prevent an adversary with control of traffic from one Network Perspective from poisoning the DNS cache used by other Network Perspectives). The network infrastructure providing Internet connectivity to a Network Perspective MAY be administered by the same organization providing the computational services required to operate the Network Perspective. All communications between a remote Network Perspective and the CA MUST take place over an authenticated and encrypted channel relying on modern protocols (e.g., over HTTPS).
+
+A Network Perspective MAY use a recursive DNS resolver that is NOT co-located with the Network Perspective. However, the DNS resolver used by the Network Perspective MUST fall within the same Regional Internet Registry service region as the Network Perspective relying upon it. Furthermore, for any pair of DNS resolvers used on a Multi-Perspective Issuance Corroboration attempt, the straight-line distance between the two States, Provinces, or Countries the DNS resolvers reside in MUST be at least 500 km. The location of a DNS resolver is determined by the point where unencapsulated outbound DNS queries are first handed off to the network infrastructure providing Internet connectivity to that DNS resolver.
```

Comment:
Do I understand this paragraph correctly in that if a remote perspective uses a DNS resolver NOT run by the CA itself, this would still be considered a Delegated Third Party and thus be forbidden?

Reply:
@romanf - One important clarification - the proposal does not prohibit Delegated Third Parties from participating in MPIC. MPIC Network Perspectives should be considered as having the potential of blocking issuance, not authorizing it (this conclusion must be derived by the Primary Network Perspective) as described in 3.2.2.4 and 3.2.2.5.

Reply:
@romanf / @tobij / @timfromdigicert 

[Sorry for the odd formatting/threading of these messages, I'm responding to these comments directly in GitHub rather than via email.]

Concerning the use of VPNs:
- My interpretation of the proposal is that use of VPNs would not be considered prohibited. 
- My position is that both 1st (CA-operated) and 3rd party (non-CA operated) VPNs should be permitted. At a network level, a VPN achieves the same effect of causing traffic egress at a different location as actually running validator code at remote vantage perspectives. While care must be taken to ensure that the DNS cache is not shared between different MPIC calls as described required in the proposal, fundamentally, the VPN implementation can achieve the exact same security properties against BGP attacks as remote validators (i.e., those hosted on a cloud VM).
- It's not clear to me how relying on a VPN provider for MPIC is meaningfully different than when considering how a CA relies on their existing Internet Service Provider for outbound internet connectivity. 

Comments and considerations welcome!

Reply:
@ryancdickson I haven't thought about it too much in detail but I think your bullet points are at least in the neighborhood of correct, and would be a great start.